_posts/2015-11-23-elm-for-javascript-developers.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Elm for JavaScript Developers
+intro: In this post we'll explore Elm, a new programming language aimed at frontend developers.
+---
+
```

Comment:
`I'll talk through some of the language's major features`

`realiise` -> `realise`